airbyte-workers/Dockerfile

```diff
@@ -1,11 +1,12 @@
-ARG JDK_VERSION=17.0.4
+ARG JDK_VERSION=17-al2023
```

Comment:
Should we do the same in all other Dockerfiles?

Reply:
Done